stacks-v3/src/App.js

```diff
@@ -506,7 +506,24 @@ class App extends Component {
     }
     return today;
   };
-  dayController = () => {};
+  dayController = () => {
+    //get the day
+    let fullDate = new Date();
+    let thisDay = fullDate.getDate().toString();
+    let thisMonth = fullDate.getMonth().toString();
+    let thisYear = fullDate.getFullYear().toString();
+
+    //build dates
+    let currentDate = thisDay + "/" + thisMonth + "/" + thisYear;
+    let visDate = this.visualDate(thisDay, thisMonth);
+
+    //check if is new day
+    if (this.isNewDay(currentDate, visDate)) {
+      this.newDayUpdateStreakCounter();
```

Comment:
This could be encapsulated to its own function:

```
if (this.isNewDay(currentDate, visDate)) {
  this.onNewDay()
}
```

As you may want this exact sequence to run from somewhere else in the application. Your thoughts?

Reply:
Makes sense!